eng/Packages.props

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <DotNetAssemblyVersion>3.1.3</DotNetAssemblyVersion>
+  </PropertyGroup>
+  <ItemGroup> 
+    <PackageReference Update="Azure.Core" Version="1.0.1" /> 
```

Comment:
Can we use version 1.0.2 here? https://github.com/dotnet/arcade-services/pull/1163

Reply:
Sure. :-)

Reply:
Done.

Reply:
on the plus side, the usage of Packages.props should aleviate a ton of the package versin conflict. Since it's now not possible for any package in either arcade-services or Helix to get out sync.  (There is a tool in this PR to sync Package.props files between repos, I don't mess around when I go for consistenct)